apps/explorer/src/components/ownedobjects/OwnedObjectConstants.tsx

```diff
@@ -1,14 +1,16 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-export type DataType = {
+// todo: remove these
+export type Data = {
     id: string;
     Type: string;
     _isCoin: boolean;
     Version?: string;
     display?: string;
     balance?: bigint;
     name?: string;
-}[];
+};
+export type DataType = Data[];
 
-export const ITEMS_PER_PAGE: number = 6;
+export const ITEMS_PER_PAGE: number = 8;
```

Comment:
Curious why we're bumping

Reply:
bumped it to 8 to align with the default in Figma: 

![image](https://user-images.githubusercontent.com/122397493/223612604-23910367-eb06-43c0-bde4-6734bb90ba00.png)

https://www.figma.com/file/kSD0qXCIDZA4KQVNepA5hR/02-Explorer?node-id=204-54037&t=qH5XGBaO26cfBdWs-4